src/frontend/app/store/effects/endpoint.effects.ts

```diff
@@ -203,6 +204,9 @@ export class EndpointsEffect {
       if (actionStrings[0]) {
         this.store.dispatch({ type: actionStrings[0], guid: apiAction.guid });
       }
+      if (endpointType === 'cloud-foundry' && apiActionType === 'connect') {
+        this.store.dispatch(new ClearPages('application', 'applicationWall'));
```

Comment:
I would dispatch the action in a new effect attached to CONNECT_ENDPOINTS_SUCCESS rather than adding a new ApiRequestTypes.